updating/index.adoc

```diff
@@ -0,0 +1,90 @@
+
+['updating-clusters-overview-upgrade-channels-and-releases']
+== Upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. For more information, check the following information:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding Fast and stable channel use and strategies]
+* xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+* xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+* xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Understanding conditional updates]
+
+['updating-clusters-overview-prepare-eus-to-eus-update']
+== Preparing to perform an EUS-to-EUS update
+xref:../updating/preparing-eus-eus-upgrade.adoc#preparing-eus-eus-upgrade[Preparing to perform an EUS-to-EUS update]: Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized. You must update from {product-title} 4.8 to 4.9, and then to 4.10. You cannot update from {product-title} 4.8 to 4.10 directly. For more information, check the following item:
+
+* xref:../updating/preparing-eus-eus-upgrade.adoc#updating-eus-to-eus-upgrade_eus-to-eus-upgrade[Updating EUS-to-EUS]
+
+['updating-clusters-overview-update-cluster-using-web-console']
+== Updating a cluster within a minor version using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, check the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-using-custom-machine-config-pools-canary_updating-cluster-within-minor[Performing a canary rollout update]
```

Comment:
[Performing a canary rollout update](https://github.com/openshift/openshift-docs/blob/6e709b6cae990abcc56d7e74a5337987f333145c/updating/updating-cluster-within-minor.adoc#update-using-custom-machine-config-pools-canary_updating-cluster-within-minor)
A wrong link here?

Reply:
Addressed

Reply:
Hi, @xenolinux My original comment was to check if we need the three sections in https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-within-minor.html. After I confirmed with each part's owner, and they think the above three sections are needed even with some doc bugs. They will open corresponding bugs for each part. And so for this index, i think we should add them back to keep the content to be consistent with the `Updating a cluster within a minor version using the web console`. Sorry for the mis-understanding here.

Reply:
no problem. I addressed this change. I kept two sections only that are present in 4.6 docs. Let me know if that needs to be changed